atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -142,6 +152,21 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji connection established")
 
+    def _fetch_log_files(self):
+        try:
+            namespace = get_build_json()['metadata']['namespace']
+        except KeyError:
+            namespace = None
+        osbs_conf = Configuration(conf_file=None, openshift_uri=self.url,
+                                  use_auth=self.use_auth, verify_ssl=self.verify_ssl,
+                                  namespace=namespace)
+        osbs = OSBS(osbs_conf, osbs_conf)
+        build_info = get_build_json()['metadata']['name'] or {}
```

Comment:
Name is actually the build "ID".
See:
https://docs.openshift.com/container-platform/3.6/rest_api/openshift_v1.html#v1-build
https://docs.openshift.com/container-platform/3.6/rest_api/openshift_v1.html#v1-objectmeta